specs/protocol/system-config.md

```diff
@@ -5,164 +5,304 @@
+transactions, this is the fixed intrinsic gas cost of the L1 transaction.
 
-The `baseFeeScalar` and `blobBaseFeeScalar` are incorporated into the L2 through the
-[Ecotone L1 attributes deposit transaction calldata](deposits.md#l1-attributes---ecotone).
+The `blobBaseFeeScalar` corresponds to the share of a user-transaction (per byte) in the total
+blobdata that is introduced by the data-transaction of the batch-submitter.
 
-Future upgrades of the `SystemConfig` contract may provide additional typed getters/setters
-for the versioned scalar information.
+### Unsafe Block Signer
 
-In Ecotone the existing `setGasConfig` function, and `scalar` and `overhead` getters, continue to function.
+The **Unsafe Block Signer** is an Ethereum address whose corresponding private key is used to sign
+"unsafe" blocks before they are published to L1. This signature allows nodes in the P2P network to
+recognize these blocks as the canonical unsafe blocks, preventing denial of service attacks on the
+P2P layer.
 
-When the batch-submitter utilizes EIP-4844 blob data for data-availability
-it can adjust the scalars to accurately price the resources:
+To ensure that its value can be fetched with a storage proof in a storage layout independent
+manner, it is stored at a special storage slot corresponding to
+`keccak256("systemconfig.unsafeblocksigner")`.
 
-- `baseFeeScalar` to correspond to the share of the user-transaction (per byte)
-  in the total regular L1 EVM gas usage consumed by the data-transaction of the batch-submitter.
-  For blob transactions this is the fixed intrinsic gas cost of the L1 transaction.
+Unlike other system config parameters, the Unsafe Block Signer only operates on blockchain policy
+and is not a consensus level parameter.
 
-- `blobBaseFeeScalar` to correspond to share of a user-transaction (per byte)
-  in the total Blob data that is introduced by the data-transaction of the batch-submitter.
+### L2 Gas Limit
 
-### `gasLimit` (`uint64`)
+The **L2 Gas Limit** defines the maximum amount of gas that can be used in a single L2 block.
+This parameter ensures that L2 blocks remain of reasonable size to be processed and proven.
 
-The gas limit of the L2 blocks is configured through the system config.
-Changes to the L2 gas limit are fully applied in the first L2 block with the L1 origin that introduced the change,
-as opposed to the 1/1024 adjustments towards a target as seen in limit updates of L1 blocks.
+Changes to the L2 gas limit are fully applied in the first L2 block with the L1 origin that
+introduced the change, as opposed to the 1/1024 adjustments towards a target as seen in limit
+updates of L1 blocks.
 
-The gas limit may not be set to a value larger than `200_000_000`. This is to ensure that the L2 blocks are fault
-provable and of reasonable size to be processed by the client software. Over time, this value will be increased.
+The gas limit may not be set to a value larger than `200,000,000`. This is to ensure that L2 blocks
```

Comment:
I think we should avoid duplicating info that belongs in `configurability.md` here.